google-analytics-setup.html

```diff
@@ -0,0 +1,91 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../google-signin/google-signin-aware.html">
+<link rel="import" href="../google-apis/google-client-loader.html">
```

Comment:
"google-apis" isn't listed as a dependency in bower.json, and even if it were, I don't see "google-client-loader.html" in that repo on the [0.8-preview branch](https://github.com/googlewebcomponents/google-apis/tree/0.8-preview).